hack/make/.detect-daemon-osarch

```diff
@@ -64,7 +64,9 @@ case "$PACKAGE_ARCH" in
 		;;
 	*)
 		DOCKERFILE="Dockerfile.$PACKAGE_ARCH"
-		TEST_IMAGE_NAMESPACE="$PACKAGE_ARCH"
+		if [ "$PACKAGE_ARCH" != "aarch64" ]; then
+			TEST_IMAGE_NAMESPACE="$PACKAGE_ARCH"
+		fi
```

Comment:
The idea here is that eventually we can remove this per-arch block entirely, right? (and possibly even use the same `Dockerfile` verbatim across arches?)

(Forgot to add this comment to my "review" hahaha :innocent:)

Reply:
Right. With multi-arch images used across all the platforms supported, IMO it does make sense to unify the `Dockerfile` across arches, consequently `TEST_IMAGE_NAMESPACE` here should be removed in the future in that case(suppose I am not missing something for this reply😂)